plugins/extractors/mysql/mysql_test.go

```diff
@@ -52,7 +54,7 @@ func TestMain(m *testing.M) {
 	}
 	// exponential backoff-retry, because the application in the container might not be ready to accept connections yet
 	retryFn := func(resource *dockertest.Resource) (err error) {
-		db, err = sql.Open("mysql", "root@tcp("+host+")/")
+		db, err = sqlutil.OpenWithOtel("mysql", "root@tcp("+host+")/", semconv.DBSystemMySQL)
```

Comment:
I don't think we need to open connection with Otel instrumentation in tests.

We can undo the similar change in test files.

Reply:
I think that would make the test invalid, since the actual code is actually instrumenting otel

Reply:
Actually this is only for confirming the container health, so should be fine without otel. Let me update this